src/app/character-details/character-details.component.spec.ts

```diff
@@ -0,0 +1,63 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
```

Comment:
I believe this is old. The `character-details` component was already in the app. Why is it in this PR?

Reply:
Not sure why that was still in, the new commit deleted that folder.